README.markdown

```diff
@@ -4,134 +4,15 @@
-
-This will start the server on port 8001 (http://localhost:8001), reload the
-application whenever changes are made to the source code, and restart it should
-it crash for some reason.
-
-**That's it!** You’ve finished setting up your lms development environment. Run
-`make help` to see all the commands that're available for running the tests,
-linting, code formatting, Python and SQL shells, etc.
-
-### Run the tests
-
-You should now be able to run the tests successfully by typing:
-
-    make test
-
-See [`HACKING.md`](HACKING.md) for documentation of common development tasks.
-
-### Bypassing the browser's "unsafe scripts" (mixed content) blocking
-
-If you use our hosted Canvas instance at <https://hypothesis.instructure.com/>
-to test your local dev instance of the app you'll get "unsafe scripts" or "mixed content"
-warnings from your browser. This is because hypothesis.instructure.com uses https but your
-local dev app, which is running in an iframe in hypothesis.instructure.com, only uses http.
-
-You'll see a blank iframe in Canvas where the app should be, along with a warning about
-"trying to launch insecure content" like this:
-
-!["Trying to launch insecure content" error](docs/images/trying-to-launch-insecure-content.png "'Trying to launch insecure content' error")
-
-If you open the browser's developer console you should see an error message like:
-
-    Mixed Content: The page at 'https://hypothesis.instructure.com/...' was loaded over HTTPS,
-    but requested an insecure form action 'http://localhost:8001/...'. This request has been
-    blocked; the content must be served over HTTPS.
-
-Fortunately you can easily bypass this mixed content blocking by your browser.
-You should also see an "Insecure content blocked" icon in the top right of the location bar:
-
-!["Insecure content blocked" dialog](docs/images/insecure-content-blocked.png "'Insecure content blocked' dialog")
-
-Click on the <samp>Load unsafe scripts</samp> link and the app should load successfully.
-
-## Overview and code design
-
-There are three presentations for developers that describe what the Hypothesis LMS app is and how it works. The **speaker notes** in these presentations also contain additional notes and links:
-
-1. [LMS App Demo & Architecture](https://docs.google.com/presentation/d/1eRMjS5B8Yja6Aupp8oKi-UztIJ9_8KRViSc6OMDLfMY/)
-2. [LMS App Code Design Patterns](https://docs.google.com/presentation/d/1AWcDoHaV9aAvInefR54SJepZiNM08Zou9jxNssccw3c/)
-3. [Speed Grader Workshop](https://docs.google.com/presentation/d/1TJF9SXRMbtHCPnkD9sy-TXe_u55--zYt6veVW0M6leA/) (about the design of the first version of our Canvas Speed Grader support)
+Hacking
+-------
 
+For how to set up the LMS development environment see
+[HACKING.md](https://github.com/hypothesis/lms/blob/main/HACKING.md).
 
+Docs
+----
 
+For other documentation see the [docs folder](https://github.com/hypothesis/lms/tree/main/docs).
```

Comment:
The `README` file is now pretty empty since there wasn't much in it except the dev env setup instructions. This makes space to possibly put some other project-specific docs in the `README` but that's beyond the scope of this PR